packages/react-instantsearch-core/src/connectors/__tests__/connectQueryRules.ts

```diff
@@ -20,29 +23,31 @@ describe('connectQueryRules', () => {
 
   describe('single index', () => {
     const indexName = 'index';
-    const context = { context: { ais: { mainTargetedIndex: indexName } } };
-    const getProvidedProps = connect.getProvidedProps.bind(context);
-    const getSearchParameters = connect.getSearchParameters.bind(context);
+    const contextValue: any = { mainTargetedIndex: indexName };
 
     describe('default', () => {
       it('without userData provides the correct props to the component', () => {
-        const props: QueryRulesProps = {
+        const props: ConnectedProps<QueryRulesProps> = {
           ...defaultProps,
+          contextValue,
```

Comment:
We can move the value inside the `defaultProps` that would avoid to repeat it N times in the file.